src/music.cpp

```diff
@@ -0,0 +1,106 @@
+#include "music.h"
+#include "debug.h"
+
+namespace music
+{
+
+std::vector<std::pair<music_id, bool>> music_id_list { std::make_pair( mp3, false ), std::make_pair( instrument, false ), std::make_pair( sound, false ), std::make_pair( title, true ) };
+
+bool is_active_music_id( music_id data )
+{
+    for( std::pair<music_id, bool> &pair : music_id_list ) {
+        if( pair.first == data ) {
+            return pair.second;
+        }
+    }
+
+    return false;
+}
+
+std::string enum_to_string( music_id data )
+{
+    switch( data ) {
+        case mp3:
+            return "mp3";
+        case instrument:
+            return "instrument";
+        case sound:
+            return "sound";
+        case title:
+            return "title";
+    }
+
+    cata_fatal( "Invalid music_id in music::enum_to_string" );
+}
+
+music_id string_to_enum( std::string data )
```

Comment:
And if it is, you then don't need to implement this.

Reply:
That was very true. I wasn't familiar with the project enough to know that this exists, so thanks for letting me know! I applied suggestion along with others in my recent commit.

Reply:
Yeah, every so often you find a super handy tool hidden somewhere, they're not exactly advertised well.

Reply:
Lol very true.